From 0972973bba482d0013221981d1c945f2886b3d00 Mon Sep 17 00:00:00 2001
From: Andrew Hayzen <ahayzen@gmail.com>
Date: Fri, 9 Feb 2018 02:06:56 +0000
Subject: [PATCH] Patch to disable DAAP preferences, to prevent attempting to
 broadcast or search

Broadcasting and search don't work as they require a running MDNS service.
For now we patch out the preferences menu, the user can still connect to a DAAP
share from the menu in the toolbar.
---
 plugins/daap/rb-daap-plugin.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/plugins/daap/rb-daap-plugin.c b/plugins/daap/rb-daap-plugin.c
index 5e5af17e8..5aa320a97 100644
--- a/plugins/daap/rb-daap-plugin.c
+++ b/plugins/daap/rb-daap-plugin.c
@@ -139,9 +139,7 @@ static void peas_gtk_configurable_iface_init (PeasGtkConfigurableInterface *ifac
 
 RB_DEFINE_PLUGIN(RB_TYPE_DAAP_PLUGIN,
 		 RBDaapPlugin,
-		 rb_daap_plugin,
-		 (G_IMPLEMENT_INTERFACE_DYNAMIC (PEAS_GTK_TYPE_CONFIGURABLE,
-						peas_gtk_configurable_iface_init)))
+		 rb_daap_plugin,)
 
 static void
 rb_daap_plugin_init (RBDaapPlugin *plugin)
-- 
2.14.1

